src/components/Media/Image/Image.tsx

```diff
@@ -34,16 +34,17 @@ export const defaultAnimatedDivQa = 'animated-div';
 const Image = (props: ImageAllProps) => {
     const {
         parallax,
-        image,
         height,
         imageClassName,
         isBackground,
         hasVideoFallback,
         video,
         fullscreen,
+        disableImageSlider,
```

Comment:
maybe enhanced name -> disableImageSliderForArrayInput ?

Reply:
I had such an idea, but the slider appears only when we pass an array to image prop, so it seems that such a name is too verbose

Reply:
You know this fact because you've seen the code, someone who hasn't seen it might think that it's always a slider being drawn and passing this property even when not an array is passed

Reply:
you're right, i renamed it